### ISSUE TITLE
Document how to append to the grammar

### ISSUE THREAD

**Issue by jokull** [mideind/ReynirPackage#23]
If there is a way to augment the grammar file for specific project contexts, that should be documented.

**Comment by sveinbjornt** [comment]
Agreed, we need much more and better documentation of the grammar. Meanwhile, you might want to take a look at Greynir's query handling modules, which extend the grammar to handle a wide range of natural-language queries. They're the ones used by Embla.

https://github.com/mideind/Greynir/blob/master/query.py
https://github.com/mideind/Greynir/tree/master/queries
